pages/surveys/index.page.js

```diff
@@ -0,0 +1,5 @@
+import Home from 'pages/index.page';
+
+const Surveys = () => <Home />;
+
+export default Surveys;
```

Comment:
Should we instead use a [redirect](https://nextjs.org/docs/api-reference/next.config.js/redirects)? 
Not sure 100% but I think it is weird if we have the same content on 2 different URL — `/` and `/surveys` 🤔 

That been said I like how clean you wrote it ^^

Reply:
Thanks for the link, done in 3739c999daf3ed824f99b52541f6c6dfc7f08236